developer-hub/src/polkadot_sdk/mod.rs

```diff
@@ -0,0 +1,126 @@
+//! [![GitHub Repo](https://img.shields.io/badge/github-polkadot-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot)
+//!
+//! [`polkadot`] is an implementation of a Polkadot client in Rust, by `@paritytech`. The Polkadot
+//! runtimes are located under the [`fellowship/runtime`](https://github.com/polkadot-fellows/runtimes) repository.
+//!
+//! > [`polkadot`] contains useful links to further learn about Polkadot, **but is in general not
+//! > part of the SDK**, as it is rarely used by developers who wish to build on top of Polkadot.
+//!
+//! #### XCM
+//!
+//! [![XCM-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-XCM-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm)
+//!
+//! [`xcm`], short for "cross consensus message", is the primary format that is used for
+//! communication between parachains, but is intended to be extensible to other use cases as well.
+//!
+//! ### Summary
+//!
+//! The following diagram summarizes how components of the Polkadot-SDK work together:
+#![doc = simple_mermaid::mermaid!("../../../docs/mermaid/polkadot_sdk.mmd")]
+//!
+//! 1. A Substrate-based chain is a blockchain composed of a "Runtime" and a "Client". As noted
+//!    above, the "Runtime" is the application logic of the blockchain, and the "Client" is
+//!    everything else. See [`reference_docs::wasm_meta_protocol`] for an in-depth explanation of
+//!    this. The former is built with [`frame`], and the latter is built with Substrate client
+//!    libraries.
+//! 2. Polkadot is itself a Substrate-based chain, composed of the the exact same two components.
+//!    The Polkadot client code is in [`polkadot`], and the Polkadot runtimes are controlled by the
+//!    [Polkadot Fellowship](https://github.com/polkadot-fellows).
+//! 3. A parachain is a "special" Substrate based chain, whereby both the client and the runtime
+//!    components have became "Polkadot-aware" using Cumulus.
+//!
+//! ## History
+//!
+//! Substrate, Polkadot and Cumulus used to each have their own repository, each of which is now
+//! archived. For historical context about how they merged into this mono-repo, see:
+//!
+//! - <https://polkadot-public.notion.site/Polkadot-SDK-FAQ-fbc4cecc2c46443fb37b9eeec2f0d85f>
+//! - <https://forum.polkadot.network/t/psa-parity-is-currently-working-on-merging-the-polkadot-stack-repositories-into-one-single-repository/2883>
+//!
+//! ## Notable Upstream Crates:
+//!
+//! - [`parity-scale-codec`](https://github.com/paritytech/parity-scale-codec)
+//! - [`parity-db`](https://github.com/paritytech/parity-db)
+//! - [`trie`](https://github.com/paritytech/trie)
+//! - [`parity-common`](https://github.com/paritytech/parity-common)
```

Comment:
Why link to the Githubs here rather than the Rust docs?

Reply:
No strong opinion. Ideally they are equivalent, one should easily lead to the other.